### PR TITLE
WIP: learning ROADMAP.md

### PR DESCRIPTION
**Pull request by facundoolano**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+1. Designing Data-intensive applications book
+1. https://bravenewgeek.com/from-the-ground-up-reasoning-about-distributed-systems-in-the-real-world/
+1. https://aphyr.com/posts/281-jepsen-on-the-perils-of-network-partitions
+1. MapReduce: Simplified Data Processing on Large Clusters 
+1. Dynamo: Amazon’s Highly Available Key-value Store
+1. Bigtable: A Distributed Storage System for Structured Data
+1. Time, Clocks and Ordering of Events in a Distributed System
+1. Unreliable failure detectors and reliable distributed systems
+
+### Cassandra
```
Comment (unbalancedparentheses) [inline ROADMAP.md]: @facundoolano what do you think about removing the cassandra links? we might added them to a more general repo related to sysdist links we recommend reading

Comment (unbalancedparentheses) [reply ROADMAP.md]: the same applies to kafka. what do you think?

Comment (facundoolano) [reply ROADMAP.md]: I agree, they're just here so we don't lose the curated list, but they belong somewhere else (not sure how that repo would look)

Comment (unbalancedparentheses) [reply ROADMAP.md]: Cool. Please @AminArria remove thoses and add this list to the general readme

**Pull request merged**